clients/native/src/commands/init.rs

```diff
@@ -107,15 +108,17 @@ async fn _prepare_temporary_credential(validators: &[Url], raw_identity: &[u8])
         .expect("could not obtain aggregate verification key of validators");
 
     let params = Parameters::new(TOTAL_ATTRIBUTES).unwrap();
-    let bandwidth_credential_attributes = BandwidthVoucherAttributes {
-        serial_number: params.random_scalar(),
-        binding_number: params.random_scalar(),
-        voucher_value: hash_to_scalar(BANDWIDTH_VALUE.to_be_bytes()),
-        voucher_info: hash_to_scalar(String::from("BandwidthVoucher").as_bytes()),
-    };
+    let bandwidth_credential_attributes = BandwidthVoucher::new(
+        &params,
+        &BANDWIDTH_VALUE.to_string(),
+        VOUCHER_INFO,
+        String::new(),
```

Comment:
Instead of those default values, I think I'll put some actual generated keys in there, even though those clients are not yet integrated with the code in the placeholder credential client.